recipes-extended/xen/files/add-xc-hypercall-page.patch

```diff
@@ -1,14 +1,12 @@
---- a/xen/arch/x86/guest/hypercall_page.S	2018-11-12 14:51:49.920565676 -0500
-+++ b/xen/arch/x86/guest/hypercall_page.S	2018-11-12 14:52:39.357054001 -0500
-@@ -59,8 +59,10 @@
- DECLARE_HYPERCALL(domctl)
- DECLARE_HYPERCALL(kexec_op)
+--- a/xen/arch/x86/guest/hypercall_page.S
++++ b/xen/arch/x86/guest/hypercall_page.S
+@@ -61,6 +61,9 @@ DECLARE_HYPERCALL(kexec_op)
  DECLARE_HYPERCALL(tmem_op)
--DECLARE_HYPERCALL(xc_reserved_op)
-+DECLARE_HYPERCALL(v4v_op)
+ DECLARE_HYPERCALL(argo_op)
```

Comment:
Will the Argo and v4v hypercalls collide?

Reply:
^ That is correct, they can live alongside each other as long as their hypercall macro numbers are separate.